settings_base.py

```diff
@@ -130,3 +130,5 @@
 # During testing, you may want to use Django's built-in static file server.
 SERVE_STATIC = False
 STATIC_BASE = 'static/'
+
+FASTLANE_USER_NAME = 'user'
```

Comment:
I wonder (but am not a Django expert) if we should move this property into `local_template/settings.py` instead, or discuss whether we need two `settings`es at all. (Note that the CIB operator creates a `local/settings.py` which they customize to their setup from `local_template/settings.py` which is included in the CIB repo).